snorkel/slicing/modules/slice_combiner.py

```diff
@@ -44,83 +44,79 @@ def __init__(
         self.slice_pred_key = slice_pred_key
         self.slice_pred_feat_key = slice_pred_feat_key
 
-    def forward(self, outputs: Dict[str, torch.Tensor]) -> torch.Tensor:  # type: ignore
+    def forward(  # type:ignore
+        self, flow_dict: Dict[str, torch.Tensor]
+    ) -> torch.Tensor:
         """Reweights and combines predictor representations given output dict.
 
         Parameters
         ----------
-        outputs
+        flow_dict
```

Comment:
Hm the description is very insider-y. Can we make it easier to parse for a deeper who isn't familiar with slicing?

Reply:
Perhaps an Example would be the best way to do it?